app/bundles/PageBundle/Controller/PublicController.php

```diff
@@ -459,10 +459,6 @@ public function redirectAction($redirectId)
         // Get query string
         $query = $this->request->query->all();
 
-        // Unset the clickthrough from the URL query
-        $ct = $query['ct'];
-        unset($query['ct']);
-
```

Comment:
OK so I found an issue we're facing. Tests are failing which is good because it allowed me to catch a regression. 

Previously, we would use these lines to prevent the `ct` from being appended back on the URL on line 468. By removing this block, line 468 _does_ append the `ct` back it on to the URL, causing unit tests to fail.

Instead of moving it in the first place, we should probably just use a null coalescing operator - https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op